ycmd/completers/general/tests/filename_completer_test.py

```diff
@@ -34,6 +34,17 @@
   'file_data' : { PATH_TO_TEST_FILE : { 'filetypes' : [ 'cpp' ] } }
 }
 
+def _CompletionResultsForLine( filename_completer, contents, extra_data=None ):
+  request = REQUEST_DATA.copy()
+  request[ 'column_num' ] = len( contents ) + 1
+  request[ 'file_data' ][ PATH_TO_TEST_FILE ][ 'contents' ] = contents
+  if extra_data:
+    request.update( extra_data )
+
+  request = RequestWrap( request )
+  candidates = filename_completer.ComputeCandidatesInner( request )
+  return [ ( c[ 'insertion_text' ], c[ 'extra_menu_info' ] )
+          for c in candidates ]
 
```

Comment:
This is copy/pasted and only changed to replace `self` with `filename_completer`

Where are you expecting another line?